docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -0,0 +1,314 @@
+2. Clone the fork on your workstation:
+
+```bash
+git clone git@github.com:{YOUR_USERNAME}/airbyte.git
+cd airbyte
+```
+
+3. Make sure that `abctl` is running correctly with the following command `abctl local status`. Verify the status check
+was successful
+
+4. Then, build the connector image:
+    - Verify the `airbyte-ci` tool is installed by running `airbyte-ci --help`, install the command with the instructions in the Prerequisites if the command is not found.
+    - Run `airbyte-ci connectors --name source-<source-name> build` will build your connector image.
+    - Once the command is done, you will find your connector image in your local docker host: `airbyte/source-<source-name>:dev`.
+
+5. You can then load the newly built image into the `abctl` instance using:
+
+```shell
+kind load docker-image airbyte/source-<source-name>:dev -n airbyte-abctl
+```
+
+:::info
+
+The above connector image is tagged with `dev`. You can change this to use another tag if you'd like.
+
+:::
+
+- In your browser, visit [http://localhost:8000/](http://localhost:8000/)
+- Log in
+- Go to `Settings` (gear icon in lower left corner)
+- Go to `Sources` or `Destinations` (depending on which connector you are testing)
+- Update the version number to use your docker image tag (default is `dev`)
+- Click `Change` to save the changes
+
+Now when you run a sync with that connector, it will use your local docker image
+
+
+### Connector Specification Caching
+
+The Airbyte Server caches connector specifications for performance reasons. If you update the specification of a 
+connector, you will need to clear this cache so the new changes are registered. To do this:
+
+- In your browser, visit [http://localhost:8000/](http://localhost:8000/)
+- Log in
+- Go to `Settings` (gear icon in lower left corner)
+- Go to `Sources` or `Destinations` (depending on which connector you are testing)
+- Update the version number to most recent release docker image tag, e.g. `1.6.4`
+- Click `Change` to save the changes
+- Set the version back to `dev`
+- Click `Change` to save the changes
+
+## Platform Contributions
+1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) the [ `airbyte-platform`](https://github.com/airbytehq/airbyte-platform) repository.
+2. Clone the fork on your workstation:
+
+```bash
+git clone git@github.com:{YOUR_USERNAME}/airbyte-platform.git
+cd airbyte-platform
+```
+
+3. Make sure that `abctl` is running correctly with the following command `abctl local status`. Verify the status check
+was successful
+
+### Build with `gradle`
+
+To compile and build the platform, run the following command in your local `airbyte-platform` repository:
+
+```bash
+./gradlew build
+```
+
+This will build all the code and run all the unit tests.
+
+`./gradlew build` creates all the necessary artifacts \(Webapp, Jars and Docker images\) so that you can run Airbyte locally. Since this builds everything, it can take some time.
```

Comment:
```suggestion
`./gradlew build` creates all the necessary artifacts \(Webapp, Jars, and Docker images\) so that you can run Airbyte locally. Since this builds everything, it can take some time.
```